### PR TITLE
Jfm/chef18.2 release docs

### PR DESCRIPTION
**Pull request by johnmccrae**
## Description

I ran the docs:update command for Chef Infra and copied that over. I also had 2 new docs which I added as well. There are a LOT of small, niggling changes to the base yaml files that I am sure are mostly crap but I don't know enough about the process to edit them out ahead of time. Guide me, Sensei!

## Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

## Check List

- [ ] Spell Check
- [ ] Local build
- [ ] Examine the local build
- [ ] All tests pass

### CODE REVIEW

File: data/infra/resources/bash.yaml [hunk 1]
```diff
@@ -128,6 +126,11 @@ properties_list:
   new_in: '16.2'
   description_list:
   - markdown: An optional property to set the input sent to the command as STDIN.
+- property: interpreter
+  ruby_type: String
+  required: false
+  description_list:
+  - markdown:
```
Comment (IanMadd) [inline data/infra/resources/bash.yaml]: Can we get some text about the interpreter property.

File: data/infra/resources/launchd.yaml [hunk 2]
```diff
@@ -125,45 +124,6 @@ properties_list:
...
-  required: false
-  description_list:
-  - markdown: 'The type of resource. Possible values: `daemon` (default), `agent`.'
-- property: abandon_process_group
-  ruby_type: true, false
-  required: false
-  description_list:
-  - markdown: 'If a job dies, all remaining processes with the same process ID may
-
-      be kept running. Set to true to kill all remaining processes.'
```
Comment (IanMadd) [inline data/infra/resources/launchd.yaml:166]: Should these all be deleted? They're still listed in the example code block. `label` and `mode` were just moved lower in the file.

File: data/infra/resources/selinux_state.yaml [hunk 3]
```diff
@@ -27,9 +27,9 @@ syntax_full_properties_list:
   available to this resource."
 actions_list:
   :nothing:
-    shortcode: resources_common_actions_nothing_default.md
+    shortcode: resources_common_actions_nothing.md
   :enforcing:
-    markdown: Set the SELinux state to enforcing.
+    markdown: Set the SELinux state to enforcing. (default)
```
Comment (IanMadd) [inline data/infra/resources/selinux_state.yaml:32]: Is this correct? Default action is enforcing?

File: data/infra/resources/service.yaml [hunk 4]
```diff
@@ -36,44 +36,48 @@ syntax_full_properties_list:
...
   :stop:
     markdown: Stop a service.
+  :restart:
+    markdown: 
+  :reload:
+    markdown: 
+  :mask:
+    markdown: 
+  :unmask:
+    markdown:
```
Comment (IanMadd) [inline data/infra/resources/service.yaml]: Looks like there's a discrepancy with the syntax example above.

**Review by IanMadd — COMMENTED** [review]
Feedback:

File: data/infra/resources/bash.yaml (hunk 1 above)
Comment (johnmccrae) [reply data/infra/resources/bash.yaml]: That turned out to be bogus. Removed!

File: data/infra/resources/launchd.yaml (hunk 2 above)
Comment (johnmccrae) [reply data/infra/resources/launchd.yaml:166]: Something is really whacky here. In my local copy of that file, in my branch, those lines are not deleted. Hmmmm...

File: data/infra/resources/selinux_state.yaml (hunk 3 above)
Comment (johnmccrae) [reply data/infra/resources/selinux_state.yaml:32]: Yes, that changed.

File: data/infra/resources/service.yaml (hunk 4 above)
Comment (johnmccrae) [reply data/infra/resources/service.yaml]: Weird. Now redacted

File: data/infra/resources/launchd.yaml (hunk 2 above)
Comment (IanMadd) [reply data/infra/resources/launchd.yaml:166]: Oh, I see what's going on. These are duplicate properties. The dups are getting deleted here. They got added in #4004.

Comment (IanMadd) [reply data/infra/resources/launchd.yaml:166]: This setup really is a nightmare.

**Review by IanMadd — APPROVED** [review]

**Pull request merged**